### PR TITLE
Fix links in workflow docs

### PR DESCRIPTION
**Pull request by logason**
Fixes broken "What's next" links in the workflow docs, based on a report from a customer: https://app.asana.com/0/1199661690807061/1202883037083484/f

Main changes in a0ae54f, only some reformatting happening in 6e498a3.

### CODE REVIEW

**Review by hugihlynsson — CHANGES_REQUESTED** [review]
We have over 30 instances of `CallToAction()` and I presume that all of them have this issue. Would it be simpler to fix this in a single place in the `<PageLink>` where we're using the path?

**Comment by logason** [comment]
> We have over 30 instances of `CallToAction()` and I presume that all of them have this issue. Would it be simpler to fix this in a single place in the `<PageLink>` where we're using the path?

As I understand it this is only an issue where `CallToAction()` is being used in `.mdx`, a functionality @JuliaSifOlafs added when working on the workflow docs 🚀 

I've confirmed that all paths in `CallToAction()` in `.mdx` files provide the full URL.

I think the complication around changing the behavior of `PageLink` is the inconsistency between `.mdx` and `.tsx`, which I'm not sure why happens. Any ideas @hugihlynsson?

**Comment by hugihlynsson** [comment]
> > We have over 30 instances of `CallToAction()` and I presume that all of them have this issue. Would it be simpler to fix this in a single place in the `<PageLink>` where we're using the path?
> 
> As I understand it this is only an issue where `CallToAction()` is being used in `.mdx`, a functionality @JuliaSifOlafs added when working on the workflow docs 🚀
> 
> I've confirmed that all paths in `CallToAction()` in `.mdx` files provide the full URL.
> 
> I think the complication around changing the behavior of `PageLink` is the inconsistency between `.mdx` and `.tsx`, which I'm not sure why happens. Any ideas @hugihlynsson?

AFAIK this is an issue on all CallToActions. If you go to https://www.avo.app/docs/ and command-click on any of the page links here you get end up on the wrong link with a not-found. If you inspect the link, it's missing the `/docs` part.

File: pages/workflow/plan.mdx [hunk 1]
```diff
@@ -85,25 +88,22 @@ There are a few things to keep in mind when doing so:
...
-              new CallToAction("/workflow/review")
-            }
-          />
-  </div>
+<div className={styles.row}>
+  <PageLink
+    image={AnalyticsManagerIcon}
+    title="2. Review suggestions in Avo"
+    description="Learn how to review suggestions in Avo"
+    callToAction={new CallToAction('https://www.avo.app/docs/workflow/review')}
```
Comment (alex-tpom6oh) [inline pages/workflow/plan.mdx:107]: Do relative paths not work?

**Comment by logason** [comment]
> AFAIK this is an issue on all CallToActions. If you go to https://www.avo.app/docs/ and command-click on any of the page links here you get end up on the wrong link with a not-found. If you inspect the `<a>`'s `href`, it's missing the `/docs` part. This is a discrepancy between where Next's javascript handling is kicking in and where the browser is actually opening a new page from the `href`.

Ah, so the problem is more widespread than I thought, thanks for spotting @hugihlynsson, I'll take a closer look here 🔍

File: components/PageLink.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { FunctionComponent } from 'react';
 
 import styles from './PageLink.module.scss';
 import Icon from './Icon';
-import Link from 'next/link';
+import Link from '../components/Link';
```
Comment (logason) [inline components/PageLink.tsx:5]: I _think_ this should do the trick @hugihlynsson, don't know of a way to reproduce on dev or Vercel. Shall we push to prod and see what happens?

File: pages/workflow/plan.mdx (hunk 1 above)
Comment (logason) [reply pages/workflow/plan.mdx]: Should now after the changes in 078d391!

**Review by hugihlynsson — APPROVED** [review]
LGTM. Let's give this a shot!

**Comment by logason** [comment]
> LGTM. Let's give this a shot!

Looks like it just works! ™️

**Pull request merged**